processor/src/main/java/com/linecorp/decaton/processor/runtime/TaskExtractor.java

```diff
@@ -16,17 +16,19 @@
 
 package com.linecorp.decaton.processor.runtime;
 
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+
 /**
- * An interface for classes extracting {@link DecatonTask} from given bytes.
+ * An interface for classes extracting {@link DecatonTask} from given record.
  * @param <T> type of task.
  */
 public interface TaskExtractor<T> {
     /**
      * Extract object of type {@link DecatonTask} from given bytes.
-     * @param bytes raw message bytes.
+     * @param record {@link ConsumerRecord} to extract task from.
      * @return object of type {@link DecatonTask}.
-     * @throws RuntimeException this method can throw arbitrary {@link RuntimeException} if given bytes is invalid.
+     * @throws RuntimeException this method can throw arbitrary {@link RuntimeException} if given record is invalid.
      * If the method throws an exception, the task will be discarded and processor continues to process subsequent tasks.
      */
-    DecatonTask<T> extract(byte[] bytes);
+    DecatonTask<T> extract(ConsumerRecord<byte[], byte[]> record);
```

Comment:
Yeah, that's what I wondered.
Giving full access to ConsumerRecord might be useful for advanced users, but at the same time it makes Decaton to couple with Kafka tightly.

I propose below strategy. WDYT?
- Creating a class to hold necessary information (e.g. ConsumedRecord? name TBD. Which will include `key: byte[], value: byte[], headers: Headers`)
- Change TaskExtractor#extract signature to `extract(ConsumedRecord record)`
    * Why not `extract(key: byte[], value: byte[], headers: Headers)`?
        - To prevent extract() signature change in the future when we want to provide more field, which need all users to rewrite their extractor impls